addons/arsenalExport/$PBOPREFIX$

```diff
@@ -0,0 +1 @@
+z\potato\addons\arsenalExport
```

Comment:
Should this be independent of the assignGear framework? They're pretty hand in hand. With that said making assign gear more complicated may not be the best strategy.....